apps/services/routers.py

```diff
@@ -262,6 +263,29 @@ async def read_request_details(
     }
 
 
+@services_router.post("/{university_id}/create-students/",
+                      name="create_students_from_file",
+                      response_model=JSENDOutSchema[Union[List[CreateStudentsOut], None]],
+                      summary="Create students from file",
+                      responses={200: {"description": "Successful create students from file response"}},
```

Comment:
maybe should rename `create_students_from_file` and similar phrases  to `create_student_list_from_file `
because this way we are creating people from file, its hard to understand

Reply:
ok